iset.mm

```diff
@@ -78885,10 +78929,23 @@ sequence rather than the limit of the offsetted sequence (see
 
   $( The empty set is not a complex number.  Note: do not use this after the
      real number axioms are developed, since it is a construction-dependent
-     property.  (Contributed by NM, 2-May-1996.) $)
+     property.  See also ~ cnm which is a related property.  (Contributed by
+     NM, 2-May-1996.) $)
   0ncn $p |- -. (/) e. CC $=
     ( c0 cc wcel cnr cxp 0nelxp df-c eleq2i mtbir ) ABCADDEZCDDFBJAGHI $.
 
+  ${
+    $d A u v x $.
+    $( A complex number is an inhabited set.  Note: do not use this after the
+       real number axioms are developed, since it is a construction-dependent
+       property.  (Contributed by Jim Kingdon, 23-Oct-2023.) $)
+    cnm $p |- ( A e. CC -> E. x x e. A ) $=
```

Comment:
This is new compared to `set.mm`, but it looks like utlimately you have not used it?

Reply:
Understood, thanks!